package.json

```
@@ -26,7 +26,7 @@
 		"unit-watch": "rimraf test/tmp && ava --watch",
 		"unit-nyan": "rimraf test/tmp && ava --tap | tnyan",
 		"unit-xunit": "rimraf test/tmp && ava --tap | tap-xunit --dontUseCommentsAsTestNames=true > test-results.xml",
-		"unit-inspect": "cross-env UI5_LOG_LVL=verbose node --inspect-brk node_modules/ava/profile.js",
+		"unit-inspect": "cross-env UI5_LOG_LVL=verbose ava debug",
```

```suggestion
		"unit-inspect": "cross-env UI5_LOG_LVL=verbose ava debug --break",
```

According to https://github.com/avajs/ava/blob/master/docs/recipes/debugging-with-chrome-devtools.md

---

Done